website/docs/best-practices/how-we-mesh/mesh-4-faqs.md

```diff
@@ -251,6 +251,14 @@ If you’re interested in beta access to “Staging” environments, let your db
 
 </detailsToggle>
 
+<detailsToggle alt_header="Does dbt Mesh work if projects are 'duplicated' (dev project <> prod project)?">
+
+The short answer is "no." Cross-project references assume that each project `name` is unique in your dbt Cloud account.
+
+There are historical limitations which required customers to "duplicate" projects, so that one actual dbt project (codebase) would map to more than one dbt Cloud project. To that end, we are working to remove the historical limitations that required customers to "duplicate" projects in dbt Cloud — Staging environments for data isolation (beta), environment-level permissions, and environment-level data warehouse connections (coming soon). Once those pieces are in place, it should no longer be necessary to define separate dbt Cloud projects simply to isolate data environments or permissions.
```

Comment:
```suggestion
Historical limitations required customers to "duplicate" projects so that one actual dbt project (codebase) would map to more than one dbt Cloud project. To that end, we are working to remove the historical limitations that required customers to "duplicate" projects in dbt Cloud — Staging environments for data isolation (beta), environment-level permissions, and environment-level data warehouse connections (coming soon). Once those pieces are in place, it should no longer be necessary to define separate dbt Cloud projects to isolate data environments or permissions.
```